firehose.c

```diff
@@ -215,9 +221,14 @@ static int firehose_configure_response_parser(xmlNode *node)
 	size_t max_size;
 
 	value = xmlGetProp(node, (xmlChar*)"value");
+	if (!value)
+		return -EINVAL;
```

Comment:
I am not sure that EINVAL will be interpretated correctly by callers here.

Reply:
Before my patch it returns also -EINVAL.

Reply:
right. sorry i missed that ;-)